### PR TITLE
Update header styling of the canvas

### PR DESCRIPTION
**Pull request by anthonydillon**

### CODE REVIEW

**Comment by jujugui** [comment]
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/2947/
Test FAILed.

**Comment by anthonydillon** [comment]
### QA
- Open the canvas and check the header matches the design: https://drive.google.com/a/canonical.com/file/d/0B2sqJRO7vy3ZcGNUR0pVSkNDQ0k/view?usp=sharing
- There is an current issue with the login and so "anonymous" is hard coded

File: jujugui/static/gui/src/app/app.js [hunk 1]
```diff
@@ -633,7 +633,7 @@ YUI.add('juju-gui', function(Y) {
         }, this);
       }
 
-      Y.one('.navbar').delegate('click', function(e) {
+      Y.one('.header-banner').delegate('click', function(e) {
```
Comment (hatched) [inline jujugui/static/gui/src/app/app.js]: You'll want to have this wrapped in a conditional with the react flag

File: jujugui/static/gui/src/app/views/utils.js [hunk 2]
```diff
@@ -1718,7 +1718,7 @@ YUI.add('juju-view-utils', function(Y) {
     var containerHeight = Y.one('body').get(
         primary ? 'winHeight' : 'docHeight'),
         bottomNavbar = Y.one('.bottom-navbar'),
-        navbar = Y.one('.navbar'),
+        navbar = Y.one('.header-banner'),
```
Comment (hatched) [inline jujugui/static/gui/src/app/views/utils.js]: flag

File: jujugui/static/gui/src/app/assets/css/_header.scss [hunk 3]
```diff
@@ -0,0 +1,101 @@
+
+/*
+ * The main header above the canvas
+ */
+
+ %list-style {
```
Comment (huwshimi) [inline jujugui/static/gui/src/app/assets/css/_header.scss]: Maybe you could chuck this in _mixins.scss as it seems pretty handy.

File: jujugui/static/gui/src/app/assets/css/_header.scss [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: $navbar-height;
+
+    .header-banner__link,
+    .header-banner__link--breadcrumb {
+        @extend .border-box;
+        color: #333;
```
Comment (huwshimi) [inline jujugui/static/gui/src/app/assets/css/_header.scss]: You could use `$text-colour` here as I've updated it to be `#333` in another branch (not yet landed, you could update `$text-colour` in base.scss to be `#333` in this branch too if you wanted).

File: jujugui/static/gui/src/app/assets/css/_header.scss [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+        display: block;
+    }
+
+    .header-banner__link--breadcrumb {
+        color: $charm-panel-orange;
+        font-size: 23px;
+
+        &:before {
+            content: '/';
```
Comment (huwshimi) [inline jujugui/static/gui/src/app/assets/css/_header.scss]: Nice way to handle this!

File: jujugui/static/gui/src/app/assets/css/_header.scss [hunk 6]
```diff
@@ -0,0 +1,101 @@
...
+            position: relative;
+            right: -5px;
+            top: 1px;
+            color: #cdcdcd;
+        }
+    }
+
+    .environment-name {
+        color:  #333333;
```
Comment (huwshimi) [inline jujugui/static/gui/src/app/assets/css/_header.scss]: Again, maybe use `$text-colour`.

File: jujugui/static/gui/src/app/store/env/sandbox.js [hunk 7]
```diff
@@ -319,7 +319,7 @@ YUI.add('juju-env-sandbox', function(Y) {
         Response: {
           ProviderType: state.get('providerType'),
           DefaultSeries: state.get('defaultSeries'),
-          Name: 'Sandbox'
+          Name: 'sandbox'
```
Comment (huwshimi) [inline jujugui/static/gui/src/app/store/env/sandbox.js]: For some reason I'm still seeing this with the uppercase S. Could just be me though.

**Comment by huwshimi** [comment]
Nice one @anthonydillon! Should the sign in button be orange? I just noticed that the Sign up button was...

File: jujugui/static/gui/src/app/store/env/sandbox.js (hunk 7 above)
Comment (anthonydillon) [inline jujugui/static/gui/src/app/store/env/sandbox.js]: That's odd, I'm seeing it in lowercase. Could you try `make clean-gui run` please?

**Comment by jujugui** [comment]
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/2954/
Test FAILed.

**Comment by jujugui** [comment]
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/2955/
Test FAILed.

**Comment by hatched** [comment]
:+1: QA OK! Looks awesome

**Comment by jujugui** [comment]
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/2957/
Test FAILed.

**Comment by jujugui** [comment]
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/2960/
Test FAILed.

**Comment by jujugui** [comment]
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/2966/
Test FAILed.

**Comment by jujugui** [comment]
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/2967/
Test FAILed.

**Comment by jujugui** [comment]
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/2968/
Test FAILed.

**Comment by jujugui** [comment]
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/2969/
Test FAILed.

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/2970/
Test PASSed.

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/2971/
Test PASSed.

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/2972/
Test PASSed.

**Comment by anthonydillon** [comment]
Thanks @huwshimi @hatched :shipit:

**Comment by hatched** [comment]
:shipit: turns out CI still doesn't like ant

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**